keyboards/designedbygg/darkmage/config.h

```diff
@@ -30,4 +30,7 @@
 /* RGB LED Config */
 #define SN32F2XX_RGB_MATRIX_ROW_PINS { A0, A2, A1, A3, A5, A4, A6, A8, A7, A9, A11, A10, A12, A14, A13}
 #define SN32F2XX_RGB_MATRIX_COL_PINS { C0, C1, C3, C4, C5, C6, C7, C8, C9, C10, C11, C12, C13, C14, C15 }
-#define RGB_MATRIX_LED_COUNT (69)
\ No newline at end of file
+#define RGB_MATRIX_LED_COUNT (69)
+
+/* Caps lock key LED Indicator */
+#define CAPS_LOCK_KEY 29
```

Comment:
```suggestion
#define CAPS_LOCK_LED_INDEX 29
```
remember, it's the LED we care about, not the key